src/ontology/ro-edit.owl

```diff
@@ -566,6 +566,7 @@ Declaration(ObjectProperty(obo:RO_0016001))
 Declaration(ObjectProperty(obo:RO_0016002))
 Declaration(ObjectProperty(obo:RO_0016004))
 Declaration(ObjectProperty(obo:RO_0017001))
+Declaration(ObjectProperty(obo:RO_0017002))
```

Comment:
Please change IRI to `obo:RO_0017005`. I made a mistake when I assigned `obo:RO_0017002`.

Reply:
RO_0017002 has been corrected to RO_0017005